src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,67 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+    @OneToOne
+    @JoinColumn(name = "hall_id")
+    private CinemaHall cinemaHall;
+
+    private LocalDate showTime;
```

Comment:
The field `showTime` should likely be of type `LocalDateTime` instead of `LocalDate`, as movie sessions typically include the specific time of day they start, not just the date. Please verify this with the requirements of your task.